west.yml

```diff
@@ -198,7 +198,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 4597b16cfedf5553cb155151e65eb994d5d0ef25
+      revision: pull/454/head
```

Comment:
is this the right PR? I don't see any Wi-Fi driver source files added to the HAL, just the CI flag which is linked to another manifest PR?

Reply:
wifi driver already merged into hal_nxp, now this links to another PR to fix the build issue, which is caused by one hostap PR still under review.

Reply:
@krish2718, we were in the process of merging things when we found that there was a build problem. So we added the appropriate fix on this side and a change so that CI will actually build this platform. We used a path that Nordic did for platforms that require a blob to do a "build only" kconfig so CI would at least catch the error.

Reply:
Sure, I was confused by the title that `Wi-Fi driver supprot added to HAL` but in that PR didn't see those changes.